packages/components/tests/dummy/app/templates/components/form/text-input.hbs

```diff
@@ -249,6 +259,44 @@
     </SG.Item>
   </Shw::Grid>
 
+  <Shw::Divider @level={{2}} />
+
+  <Shw::Text::H3>Character count</Shw::Text::H3>
```

Comment:
What do you think of having an extra use case added to the showcase, where we connect the error with the callback function, to have the full cycle of functionality tested as we were consumers, to make sure we're not missing something (potentially, all the "counter" demos for text-input/textarea could be hooked to a validation, if that's not too complex, so we can test different scenarios)?